.github/workflows/tests.yml

```diff
@@ -237,19 +237,32 @@ jobs:
         if: always()
         run: |
           touch bottles/steps_output.txt
-          cat bottles/steps_output.txt
+
+          echo "# Build failure summary on ${{ matrix.runner }}" >> "$GITHUB_STEP_SUMMARY"
+          echo '```' >> "$GITHUB_STEP_SUMMARY"
+          cat bottles/steps_output.txt | tee -a "$GITHUB_STEP_SUMMARY"
+          echo '```' >> "$GITHUB_STEP_SUMMARY"
```

Comment:
This feels like something `brew test-bot` should do for us. We'll have to `>> "$GITHUB_STEP_SUMMARY"` here but the rest actual formatting and summary feels like something for it.

Reply:
`GITHUB_STEP_SUMMARY` is an environment variable that's meant to point to a file that contains some markdown.

I'm thinking this variable should be visible to `test-bot` (unless we filter it), in which case `test-bot` can write to this file directly with the formatted content.

This should make it simpler to avoid tripping up on file permissions too (though I suspect the actual correct fix here is to work out why we don't have write permissions).

Thoughts?

Reply:
Yeh, moving to `test-bot` seems nicer. It's unfiltered by default iirc.